### PR TITLE
Publish

### PR DESCRIPTION
**Pull request by joehand**
Works!

Need to think a bit about how and what to use from `dat.json` files here. Depends on [dat-node 1 PR](https://github.com/datproject/dat-node/pull/50) & PR #28 

## TODO

* Test that it sends the correct secured request to our server.

### CODE REVIEW

**Comment by okdistribute** [comment]
I think that the `dat.json` file should just be metadata about the resource, which could include it's url when its published

File: lib/commands/auth/logout.js [hunk 1]
```diff
@@ -4,33 +4,21 @@ var TownshipClient = require('../../township')
...
-  ]
+  options: []
 }
 
 function logout (opts) {
   var client = TownshipClient({
     server: opts.server,
     config: {
-      filename: opts.config
+      filepath: opts.config
```
Comment (okdistribute) [inline lib/commands/auth/logout.js]: +1

File: lib/commands/auth/whoami.js [hunk 2]
```diff
@@ -4,25 +4,18 @@ var TownshipClient = require('../../township')
...
   })
-  var config = client.config.get()
-  var login = config.currentLogin
-  if (!login) process.stdout.write('Not logged in.')
-  else process.stdout.write(login.email)
-  ui.exit()
+  var login = client.getLogin()
+  if (!login.token) return ui.exitErr('Not logged in.')
+  console.log(login.email)
+  process.exit(0)
```
Comment (okdistribute) [inline lib/commands/auth/whoami.js:15]: does this PR have some commits from the auth cleanup PR?

**Comment by joehand** [comment]
This PR got messy going to close and open new one.

**Pull request closed without merging**